### PR TITLE
[Enhanced function] Issue #1046

### PR DESCRIPTION
**Pull request by ChengYen-Tang**
https://github.com/dotnet/TorchSharp/issues/1046

### CODE REVIEW

File: src/TorchSharp/Scalar.cs [hunk 1]
```diff
@@ -248,17 +259,18 @@ public static Scalar ToScalar(this bool value)
             return new Scalar(THSTorch_bool_to_scalar(value));
         }
 
-#if NET6_0_OR_GREATER
+#if NET5_0_OR_GREATER
```
Comment (NiklasGustafsson) [inline src/TorchSharp/Scalar.cs:262]: We no longer support .NET 5

File: src/TorchSharp/Tensor/Tensor.Factories.cs [hunk 2]
```diff
@@ -2104,6 +2103,112 @@ public static Tensor tensor(long[,,,] rawArray, ScalarType? dtype = null, Device
             return _tensor_i64(rawArray, stackalloc long[] { rawArray.GetLongLength(0), rawArray.GetLongLength(1), rawArray.GetLongLength(2), rawArray.GetLongLength(3) }, dtype, device, requires_grad, names: names);
         }
 
+#if NET5_0_OR_GREATER
```
Comment (NiklasGustafsson) [inline src/TorchSharp/Tensor/Tensor.Factories.cs:2106]: See previous comment about .NET 5

**Review by NiklasGustafsson — CHANGES_REQUESTED** [review]

**Comment by NiklasGustafsson** [comment]
Awesome! Thanks for the help. Only comment is that we no longer support .NET 5, so is should probably be `if NET5_OR_GREATER`

**Comment by ChengYen-Tang** [comment]
Ok, I will fix it tomorrow, I am a little tired today, good night.

**Review by NiklasGustafsson — APPROVED** [review]

**Pull request merged**